homeassistant/components/music_assistant/media_player.py

```diff
@@ -102,14 +102,14 @@
 
 
 def catch_musicassistant_error[_R, **P](
-    func: Callable[..., Awaitable[_R]],
-) -> Callable[..., Coroutine[Any, Any, _R | None]]:
+    func: Callable[Concatenate[MusicAssistantPlayer, P], Coroutine[Any, Any, _R]],
+) -> Callable[Concatenate[MusicAssistantPlayer, P], Coroutine[Any, Any, _R]]:
     """Check and log commands to players."""
 
     @functools.wraps(func)
     async def wrapper(
         self: MusicAssistantPlayer, *args: P.args, **kwargs: P.kwargs
-    ) -> _R | None:
+    ) -> _R:
         """Catch Music Assistant errors and convert to Home Assistant error."""
         try:
             return await func(self, *args, **kwargs)
```

Comment:
Either the return value of `await func(...)` is returned or an exception raised. Thus there isn't a need for an explicit `None` return type.